### ISSUE TITLE
Document overflow attribute and overflow conditions

### ISSUE THREAD

**Issue by cathyxz** [ampproject/amphtml#24374]
Follow up to https://github.com/ampproject/amphtml/pull/24290. 

We have automatic truncation of elements with overflowing contents and an optional overflow element that allows people to provide a fallback (a button that resizes elements on click). The heuristics for when the element gets truncated (and overflow button is shown) is not clear. We should document somewhere that resizing is allowed when: 
1. The bottom of the element is below the viewport, or
2. If the bottom of the element is within 15% of the height of the entire page AND within 1000px of the end of the page.

**Comment by CrystalOnScript** [comment]
@cathyxz I'm a little confused about this issue:

-  `the <amp-list> element can contain an element with an overflow attribute`: we don't link out to an AMP specific overflow attribute - is this referring to the CSS overflow Property? 

- The example for [specifying an overflow](https://amp.dev/documentation/components/amp-list/?format=websites#specifying-an-overflow) doesn't seem to be doing anything? Am I missing something or is it broken?

**Comment by cathyxz** [comment]
Yup. `overflow` is actually an AMP-specific property that applies to more than just amp-list. It also applies to for example, amp-iframe, amp-truncate-text, and possible some ads (though I'm not sure about this). The example you referenced doesn't do anything because in its case, there's nothing below the amp-list, so it satisfies condition 2 above, and therefore allows resizing and does not show the amp-list. If you style the overflow element to be more obvious, and also put an empty div with height 100vh below the list, it will truncate the list and show an overflow attribute. See https://codepen.io/cathyxz/pen/KKPZawQ for example.

**Comment by CrystalOnScript** [comment]
Thanks for clarifying @cathyxz 

After further investigation, the sample is displaying the "see more" button, but it's hard to see as it's placed on top of the first image. 

Are these two conditions only relevant for `amp-list` or does this apply to all uses of `overflow`?

**Comment by cathyxz** [comment]
Yup, it needs to be styled in a more visible way. I believe it applies to all uses of overflow. 
@jridgewell can keep me honest on that. =)

**Comment by jridgewell** [comment]
Yup. If an any amp element has its resize request denied by runtime, we make the overflow element appear.

**Comment by CrystalOnScript** [comment]
Hopefully my last question: @cathyxz in the two conditions when you refer to `the element` is that the element with the `overflow` attribute or the AMP component?

**Comment by cathyxz** [comment]
The element refers to the element requesting the resize, which is the AMP Component, not the element with the overflow attribute. We can think of it as allowing the developer to add an optional resize button to rectify the situation in the case that their component gets cut off due to AMP's resizing rules.

**Comment by cathyxz** [comment]
This issue is resolved by #24445.
